### PR TITLE
added system properties support for ep

### PR DESCRIPTION
**Pull request by tdt17**

### CODE REVIEW

**Comment by tdt17** [comment]
@moritzzimmer ok for you?

File: src/main/groovy/de/weltn24/gradle/plugins/JavaConventionsPlugin.groovy [hunk 1]
```diff
@@ -209,13 +209,17 @@ class JavaConventionsPlugin implements Plugin<Project> {
         project.task('smokeTest',
             type: Test,
             group: 'verification',
-            description: 'Runs the smoke tests against $SMOKETEST_SUT (must be set)') {
+            description: 'Runs the smoke tests against $SMOKETEST_SUT or stage (must be set)') {
 
             testClassesDir = project.sourceSets.smokeTest.output.classesDir
             classpath = project.sourceSets.smokeTest.runtimeClasspath
+
+            //setting JVM properties for ep (gradle forwarding)
```
Comment (moritzzimmer) [inline src/main/groovy/de/weltn24/gradle/plugins/JavaConventionsPlugin.groovy]: remove that comment

**Comment by moritzzimmer** [comment]
Fine for me, but new property should be documented in readme

**Comment by tdt17** [comment]
ready for merging

**Comment by moritzzimmer** [comment]
LGTM

**Pull request merged**